docs/topics/testing/tools.txt

```diff
@@ -384,6 +384,35 @@ Use the ``django.test.Client`` class to make requests.
         :meth:`~django.contrib.auth.models.UserManager.create_user` helper
         method to create a new user with a correctly hashed password.
 
+    .. method:: Client.force_login(user, backend=None)
+
+        .. versionadded:: 1.9
+
+        If your site uses Django's :doc:`authentication
+        system</topics/auth/index>`, you can use the ``force_login()`` method
+        to simulate the effect of a user logging into the site. This is similar
+        to the :meth:`~django.test.Client.login()` method, but this method
+        skips the authentication and verification steps of ``login()``; instead
+        it immediately logs the user in.
+
+        Inactive users (:attr:`is_active=False
+        <django.contrib.auth.models.User.is_active>`) are permitted to login
+        with this method.
+
+        This method is preferred when a test requires a user be logged in, but
+        the details of how a user logged in are not important for the purposes
+        of the test. Logging a user in with ``login()`` may require setting up
+        and keeping track of the user's credentials. Using ``force_login()``
+        simplifies this process.
+
+        Additionally, password hashing algorithms are intentionally expensive
+        to combat brute force attacks. Using this method instead of ``login()``
+        may make your tests quicker to run by skipping these algorithms.
+
+        Please note, testing and verifying the authentication system of your
```

Comment:
I guess in particular it's that if you're not using Django's built in auth system then you should test your code? I wouldn't mind if this note went away though.